h/static/scripts/account/test/auth-controller-test.coffee

```diff
@@ -18,15 +18,36 @@ class MockSession
     $promise:
       finally: sandbox.stub()
 
-mockFlash = info: sandbox.spy()
-mockFormRespond = sandbox.spy()
-
-describe 'h:AuthController', ->
+# Return an AuthController instance and associated objects for tests to use.
+_controller = (mocksession) ->
+  controller = null
   $scope = null
   $timeout = null
-  auth = null
   session = null
-  $controller = null
+  mockFlash = info: sandbox.spy()
+  mockFormRespond = sandbox.spy()
+  inject(($controller, $rootScope) ->
+    session = mocksession or new MockSession()
+    $scope = $rootScope.$new()
+    $timeout = sandbox.spy()
+    params =
```

Comment:
Thanks. I really wanted to just pass an inline anonymous object literal to the function call below, but it looks like there isn't a way to do this on CoffeeScript (unless the object literal fits on one line)? Will rename to `locals`